Packs/CortexXDR/ReleaseNotes/4_10_42.md

```diff
@@ -0,0 +1,20 @@
+
+#### Classifiers
+
+##### Cortex XDR Incident Handler - Classifier
+
+- Updated the classifier to classify Port Scan alert as XDR Incident.
+
+#### Playbooks
+
+##### Cortex XDR - Port Scan
+
+- Deprecated. Use the Cortex XDR - Port Scan - Adjusted playbook instead.
+
+##### Cortex XDR Alerts Handling
+
+- Fixed an issue in the condition of the category classification task - changed the condition to look for empty string instead of `null`
```

Comment:
what is that change? from empty string to null? I see only the ignored case-sensitive value change

Reply:
I mixed the fixes a bit, but I updated the release notes to match the fixes correctly.
One fix I made in the classifier where the condition was looking for `null` and was changed to `empty`
and the other fix was in the `Cortex XDR Alerts Handling` playbook conditional task which I added `ignorecase`